lib/whitehall/publishing_api.rb

```diff
@@ -55,6 +55,10 @@ def self.publish_gone(base_path)
       PublishingApiGoneWorker.perform_async(base_path)
     end
 
+    def self.discard_draft_async(edition)
+      PublishingApiDiscardDraftWorker.perform_async(edition.content_id, edition.translations.map(&:locale))
```

Comment:
We wanted the worker to work on all the locales at once, so as to try to keep them at least a little bit atomic.  It's a debatable opinion and I'm not sure it really warrants either consistency or documentation as it depends on the sort of update you're doing (deletes are fast and not easily prone to error).